### DISCUSSION TITLE
Use as a svg creator

### DISCUSSION THREAD

**Discussion by patolax** [vvvv/SVG#975] [Q&A]
I went through the documentation but I am not sure if this library can be used to create a svg from scratch. 

My requirement is to create a svg file from incoming structured data. Say you receive an array of letters and numbers and these needs to be place inside a sudoku board and generate a png if possible. 

Can this be done using the library. If so where can I find sample code?

**Comment by H1Gdev** [comment]
I hope this sample code will be useful.
http://svg-net.github.io/SVG/doc/Samples.html#xmloutputtester

**Comment by HazelSh** [comment]
It absolutely can be used this way, although I find the API slightly clunky for that purpose.

 You can create a new SvgDocument, add new SvgElements to it's children, then call SvgDocument.write() to create your SVG file.

The code for the example is here:
https://github.com/svg-net/SVG/blob/master/Samples/XMLOutput/Form1.cs
